merino/curated_recommendations/protocol.py

```diff
@@ -84,6 +85,7 @@ class SectionConfiguration(BaseModel):
     sectionId: str
     isFollowed: bool
     isBlocked: bool
+    followedAt: datetime | None = None
```

Comment:
I'm curious how FastAPI documents this type in [localhost:8000/docs](http://localhost:8000/docs). The docs should make clear what format and timezone `followedAt` is expected to be. Do we require UTC time? Or accept any timezone? Consider adding a description:
```suggestion
    followedAt: datetime | None = Field(
        default=None,
        description="Timestamp when the section was followed. Must be in ISO 8601 format with timezone, e.g. '2024-03-24T12:34:56Z'."
    )
```
(Untested code.)